desktop/mac/troubleshoot.md

```diff
@@ -77,7 +77,7 @@ Before you get started, we recommend that you sign into your Docker Desktop appl
 6. If you have a paid Docker subscription, click **Contact Support**. This opens the [Docker Desktop support](https://hub.docker.com/support/desktop/){:target="_blank" rel="noopener" class="_"} form. Fill in the information required and add the ID you copied earlier to the Diagnostics ID field. Click **Submit** to request Docker Desktop support.
    > **Note**
     >
-    > You must be signed in to Docker Desktop using your Pro, Team, or Business tier credentials to access the support form. For information on what's covered as part of Docker Desktop support, see [Support](#support).
+    > You must be signed in to Docker Desktop using your Pro, Team, or Business tier credentials to access the support form. For information on what's covered as part of Docker Desktop support, see [Support](../../support/index.md).
 7. If you don't have a paid Docker subscription, you can click **Upgrade to benefit from Docker Support** to upgrade your existing account.
     Alternatively, click **Report a Bug** to open a new Docker Desktop issue on GitHub. This opens Docker Desktop [for Mac](https://github.com/docker/for-mac/issues/) on GitHub in your web browser in a 'New issue' template. Complete the information required and ensure you add the diagnostic ID you copied earlier. Click **submit new issue** to create a new issue.
```

Comment:
Looks like this section could use a bit of a cleaning up;

- I think we removed the "Report a bug" _button_ (the section is still there, with a link to GitHub)
- Perhaps we should remove the link to GitHub from the docs here, and describe what happens (and make it more "generic", not "for linux", "for mac", "for windows")

Reply:
cc @aevesdocker

Reply:
@thaJeztah yes this page is on my to-do list. I'm currently tackling the dashboard page

cc @usha-mandya